fbbt/src/ontology/fbbt-edit.obo

```diff
@@ -68686,6 +68681,91 @@ relationship: overlaps FBbt:00007053 ! lateral horn
+is_a: FBbt:00003361 ! indirect flight muscle
+created_by: CP
+creation_date: 2018-06-14T08:04:07Z
+
+[Term]
+id: FBbt:00047244
+name: external muscle of fourth axillary hg1
+def: "Direct flight muscle that inserts into the fourth axillary (posterior notal wing process) and functions in carrying out lateral tip movements (Wisser and Nachtigall, 1984). It shares a tendon with the external muscle of fourth axillary hg4 (Williams and Williams, 1943). In contrast to most other wing muscles, hg1 is larger in males than in females, which is thought to allow for louder sine song (Shirangi et al., 2013). It is one of only two flight muscles that express doublesex (Shirangi et al., 2013)." [doi:10.1007/BF00312139, FlyBase:FBrf0006004, FlyBase:FBrf0098527, FlyBase:FBrf0223321]
+comment: This muscle is not illustrated in Miller 1950 (FBrf0007735).
+synonym: "hg1 muscle" EXACT []
+synonym: "posterior notal wing process muscle hg1" EXACT [FlyBase:FBrf0098527]
+is_a: FBbt:00003387 ! axillary muscle
+relationship: expresses http://flybase.org/reports/FBgn0000504
+relationship: synapsed_by FBbt:00047246 ! hg1 motor neuron
+created_by: CP
+creation_date: 2018-06-14T12:35:14Z
+
+[Term]
+id: FBbt:00047245
+name: external muscle of fourth axillary hg4
+def: "A very small muscle that lies laterally to the two internal muscles of the fourth axillary and inserts into the fourth axillary (posterior notal wing process). It shares a tendon with the external muscle of fourth axillary hg1 (Williams and Williams, 1943)." [doi:10.1007/BF00312139, FlyBase:FBrf0006004, FlyBase:FBrf0098527]
+comment: This muscle is not illustrated in Miller 1950 (FBrf0007735).
+synonym: "hg4 muscle" EXACT []
+synonym: "posterior notal wing process muscle hg4" EXACT [FlyBase:FBrf0098527]
+is_a: FBbt:00003387 ! axillary muscle
+created_by: CP
+creation_date: 2018-06-14T12:54:12Z
+
+[Term]
+id: FBbt:00047246
+name: hg1 motor neuron
+def: "Motor neuron that innervates the external muscle of fourth axillary hg1. It is required in males for sine song, but has no function in pulse song (Shirangi et al., 2013)." [FlyBase:FBrf0223321]
+is_a: FBbt:00005123 ! motor neuron
+relationship: expresses http://flybase.org/reports/FBgn0004652 ! fruitless
+relationship: synapsed_to FBbt:00047244 ! external muscle of fourth axillary hg1
+created_by: CP
+creation_date: 2018-06-14T13:45:58Z
+
+[Term]
+id: FBbt:00047247
+name: flight muscle motor neuron
+def: "." []
+intersection_of: FBbt:00005123 ! motor neuron
+intersection_of: synapsed_to FBbt:00003360 ! flight muscle
+relationship: has_soma_location FBbt:00111084 ! cell body rind of mesothoracic neuromere
+created_by: CP
+creation_date: 2018-06-14T13:59:26Z
+
+[Term]
+id: FBbt:00047248
+name: ps1 motor neuron
+def: "Motor neuron that innervates the mesothoracic pleurosternal muscle 59 (pleurosternal 1). This cell plays a role in pulse carrier frequency and pulse amplitude (Shirangi et al., 2013)." [FlyBase:FBrf0223321]
+is_a: FBbt:00005123 ! motor neuron
```

Comment:
Please add
subClassOf: 'capable of' some 'male courtship behavior, veined wing generated song production' (GO:0045433)